js/cli/src/java.ts

```diff
@@ -26,9 +29,11 @@ export const runJavaProcess = (
     "-classpath",
     classpath,
     ...javaArgs,
+    ...proxyArgs,
     mainClass,
     ...mainClassArgs
   ];
+  logger.info(`allArgs=${allArgs.join(" ")}`);
```

Comment:
wouldn't info show a bit much by default?

Reply:
forgot to remove it, but I also just realized there are a few other issues